src/Cli/ConfigGenerator.cs

```diff
@@ -82,52 +83,51 @@ public static bool TryCreateRuntimeConfig(InitOptions options, out string runtim
                         return false;
                     }
 
-                    if (!File.Exists(graphQLSchemaPath))
+                    if (!fileSystem.File.Exists(graphQLSchemaPath))
                     {
                         _logger.LogError($"GraphQL Schema File: {graphQLSchemaPath} not found.");
                         return false;
                     }
 
                     // If the option --rest.path is specified for cosmosdb_nosql, log a warning because
                     // rest is not supported for cosmosdb_nosql yet.
-                    if (!GlobalSettings.REST_DEFAULT_PATH.Equals(restPath))
+                    if (!RestRuntimeOptions.DEFAULT_PATH.Equals(restPath))
                     {
-                        _logger.LogWarning("Configuration option --rest.path is not honored for cosmosdb_nosql since " +
-                            "it does not support REST yet.");
+                        _logger.LogWarning("Configuration option --rest.path is not honored for cosmosdb_nosql since it does not support REST yet.");
                     }
 
                     restPath = null;
-                    dbOptions = new CosmosDbNoSqlOptions(cosmosDatabase, cosmosContainer, graphQLSchemaPath, GraphQLSchema: null);
+                    dbOptions.Add("database", JsonSerializer.SerializeToElement(cosmosDatabase));
```

Comment:
Can use constants for database,schema etc.

Reply:
Yep, I'll see how we can best get the from the `record` types that we have for the options.